core/components/molecules/modal/__stories__/style.css

```diff
@@ -0,0 +1,3 @@
+.Modal-list {
+    height: 185px;
+}
```

Comment:
@Rashi-Gupta-2000 Please add new line at EOF.

Reply:
in my vscode there is an extra line of code and changes reflect if I remove it but for some reason here the extra line is not visible